autoinstall/FreeBSD/14/installerconfig

```diff
@@ -92,6 +101,11 @@ else
     packages_to_install='open-vm-tools-nox11 wget curl e2fsprogs iozone lsblk'
 fi
```

Comment:
Since you defined  above `failed_packages`, I think you don't need to separate packages installing from cdrom and online repo. You can define all packages in line 67~71, and then install failed packages from online repo.